src/Core/API/addMovieAPI.js

```diff
@@ -0,0 +1,16 @@
+import api from "./axiosInstance";
+
+const addMovieAPI = async (payload) => {
+    const movie = payload.payload;
+    delete movie.url;
+
+    return api.post('/movies', {
+        ...movie
+    })
+    .then((res) => {
+        console.log(res)
+        return res.data;
+    })
```

Comment:
Don't forget about `.catch()` block for all the promises to be able to "handle" the errors.
It could contain some error logging logic.